### PR TITLE
fix/side-menu

### PR DESCRIPTION
**Pull request by markmead**
This PR fixes the side menu rendering when the page refreshes.

### CODE REVIEW

File: packages/admin/resources/views/partials/navigation/side-menu.blade.php [hunk 1]
```diff
@@ -3,7 +3,8 @@
          :class="{
              'w-64': showExpandedMenu,
```
Comment (alecritson) [inline packages/admin/resources/views/partials/navigation/side-menu.blade.php:4]: Looks like we have a custom class to handle it now, should those be reflected here?

Comment (markmead) [reply packages/admin/resources/views/partials/navigation/side-menu.blade.php:4]: It's still needed, there's no custom class being used.

Comment (alecritson) [reply packages/admin/resources/views/partials/navigation/side-menu.blade.php:4]: @markmead  What's `app-sidemenu-expanded` then?

Comment (markmead) [reply packages/admin/resources/views/partials/navigation/side-menu.blade.php:4]: It's added to the `<html>` so we can target the side menu in CSS.

Then once Alpine JS inits, we remove it and allow Alpine JS to manage the size of the side menu.

**Comment by alecritson** [comment]
@markmead Looks reasonable I think and much simpler @glennjacobs Just need your nod on this

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**